lib/cocoapods/sandbox.rb

```diff
@@ -352,7 +350,7 @@ def remove_checkout_source(name)
     #         The name of the Pod.
     #
     # @param  [#to_s] path
```

Comment:
does this need to change into `[Pathname, String]`?

Reply:
yeah I think so, good catch